src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -120,18 +137,46 @@ public async Task StopAsync(CancellationToken cancelToken = default)
             }
         }
 
-        private int GetRandomPort()
+        private int GetLikelyAvailablePort()
```

Comment:
Probably best to call this `GetAvailablePort()` since we've already checked its availability prior to returning.

Reply:
Fair enough.